pom.xml

```diff
@@ -21,6 +21,7 @@
   <properties>
     <java.version>17</java.version>
     <testcontainers.version>1.18.3</testcontainers.version>
+    <ontology-tag>v2.1.12-RC</ontology-tag>
```

Comment:
Why not `ontology.version`? And if so, I would remove the `v` and place it directly into the URL. So it is the same format as other versions.